src/main.h

```diff
@@ -341,7 +341,22 @@ bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime);
  */
 bool CheckFinalTx(const CTransaction &tx, int flags = -1);
```

Comment:
Agreed.  I was going for minimal changes to existing consensus code, but if people like I'd be happy to rename that.  Perhaps `LocktimeLock()` and `CheckLocktimeLock()`

I'd also like to take out the way CheckFinalTx is called with default -1 in the wallet code, but I could save that for another pull.